### DISCUSSION TITLE
weird test results, training only on forces

### DISCUSSION THREAD

**Discussion by benshpiro** [deepmodeling/deepmd-kit#2387] [Q&A]
Hi, I am running a test using deempd and lammps. 
I run a classic MD simulation on lammps, produce 1000 images of my system (coordinates of 20 water molecules and the forces)
and build the necessary .npy files using this data.
I then train the NN using Deepmd on these coordinates and forces, reaching a rmse of ~1eV/A for the force.  However, when running a MD simulation using lammps with the deepmd "pair style", I am not managing to reproduce the classical MD (the systems coordinates make no sense, energy is more than 2 orders of magnitude lower).
Has anyone tried this type of simple test and has insight?

**Comment by njzjz** [comment]
~1eV/A is quite large. Please check the documentation [here](https://docs.deepmodeling.com/projects/deepmd/en/master/troubleshooting/precision.html#why-does-a-model-have-low-precision).
